v4/source/users.yaml

```diff
@@ -1230,7 +1230,44 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+  
+  /users/tokens:
+    get:
+      tags:
+        - users
+      summary: Get all user access tokens for all users
+      description: |
+        Get a list of all the user access tokens for all users. Does not include the actual authentication tokens. Use query paremeters for paging.
```

Comment:
And this to 'Get a page of user access tokens for users on the system. Does not include the actual authentication tokens. Use query parameters for paging.'